nav2_map_server/package.xml

```diff
@@ -7,22 +7,22 @@
     Refactored map server for ROS2 Navigation
   </description>
   <maintainer email="brian.wilcox@intel.com">Brian Wilcox</maintainer>
-  <license>Apache License 2.0</license>
+  <license>BSD License 2.0</license>
```

Comment:
```suggestion
  <license>BSD</license>
```

No version on the BSD tag. Just the 3 letters

Reply:
Done, though I'm not sure I agree. "BSD" does not uniquely specify a license.
https://opensource.org/licenses/alphabetical

Reply:
Reverting that commit. "BSD License 2.0" is the specific identifier in ament_copyright
https://github.com/ament/ament_lint/blob/3714573db303b4d3d815f433b3ace1718df2ed2b/ament_copyright/ament_copyright/licenses.py#L41

Reply:
Ok. My mistake. I saw it was just "BSD" in the original